app/views/records/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<nav>
+  <%= link_to raw('&#8592;'), groups_path, class: 'back-btn' %>
+  <h4>EXPENSE RECORD DETAILS</h4>
+  </nav>
+<div class="container">
+<div class="record-head">
+<span style="font-size:1.2rem;font-family: 'ProximaNovaLight', sans-serif;color: #686e76;"><%= image_tag (@group.icon || 'icons/01.jpg'), alt: @group.icon, class: "icon" %>
+<%= @group.name.capitalize%></span>
+<span style="font-size:1.2rem;font-family: 'ProximaNovaBold', sans-serif;color: #686e76;
+">TOTAL:$
+<%= @group.total_amount %></span>
+</div>
+
+<ul class="list-group" style="margin-top:2%;">
+<% @records.each do |record| %>
+  <li class="list-group-item" style="display:flex;justify-content:space-between;">
+   <span style="font-size:1.1rem;color: #686e76;"><%= record.name.capitalize %></span>
+    <span class="total-amount" style="font-family: 'ProximaNovaBold', sans-serif;color: #686e76;font-size:1.1rem">$<%= record.amount %></span>
```

Comment:
- [ ] Here as per the design guideline, we have to show the transaction creation date as well. Please do that.

### Your design
![Screenshot from 2023-09-23 04-00-35](https://github.com/binodbhusal/BudgetApp/assets/26767383/5f4b962d-630b-4430-a192-67c9a42c76ca)


### Given design


![Screenshot from 2023-09-23 04-01-50](https://github.com/binodbhusal/BudgetApp/assets/26767383/8771f010-0e15-4123-bf41-25d555485b39)